sources/platform/integrations/programming/api.md

```diff
@@ -54,6 +52,10 @@ By default, tokens can access all data in your account. If that is not desirable
 
 **A scoped token can access only those resources that you'll explicitly allow it to.**
 
+:::info
+We currently do not allow scoped tokens to create or modify Actors. If you do need to create or modify Actors via Apify API, use an unscoped token.
```

Comment:
I'm really not a fan of saying "currently" cause the reader has no way of knowing what is "currently" unless they take a dive into git commit history. 
```suggestion
Scoped tokens cannot create or modify Actors. To create or modify Actors through the Apify API, use an unscoped token.
```

Reply:
Fair enough :+1:  Dropped "currently" and slightly modified the wording, taking inspiration from you.

Reply:
Fair enough 👍 Dropped "currently" and slightly modified the wording, taking inspiration from you.